source/TextBlade.ConsoleRunner/Game.cs

```diff
@@ -109,7 +112,7 @@ private void ApplyResultsIfBattle(ICommand command)
         
         var dungeonSaveData = new Dictionary<string, object>
         {
-            { "CurrentFloor", dungeon.CurrentFloorNumber },
+            { "CurrentFloor", dungeon?.CurrentFloorNumber??0 },
```

Comment:
Same thought here; this code only executes if there is a dungeon/battle.

Reply:
Same answer, really :)